src/components/CanvasWrapper.tsx

```diff
@@ -250,44 +534,17 @@ export default function CanvasWrapper(inputProps: CanvasWrapperProps): ReactElem
     };
   }, [props.dataset, canv]);
 
-  // Respond to window resizing
-  useEffect(() => {
-    /**
-     * Update the canvas dimensions based on the current window size.
-     * TODO: Margin calculation?
-     */
-    const setSize = (): void => {
-      // TODO: Potentially unsafe calculation here when using `window.innerWidth`. If close to the breakpoint where the side
-      // panel gets wrapped to below the canvas, the scrollbar added to account for the increased page height
-      // will cause this calculation to change (window.innerWidth will become smaller by ~15 pixels).
-      // Under certain circumstances, this can cause a flickering effect as the canvas resizes to accommodate the scrollbar,
-      // which causes the page to shrink, which causes the scrollbar to disappear, and so on in a loop.
-      // I've fixed this for now by setting the breakpoint to 1250 pixels, but it's not a robust solution.
-
-      // TODO: Calculate aspect ratio based on the current frame?
-      const width = Math.min(
-        containerRef.current?.clientWidth ?? props.maxWidthPx,
-        props.maxWidthPx,
-        props.maxHeightPx * ASPECT_RATIO
-      );
-      const height = Math.floor(width / ASPECT_RATIO);
-      canv.setSize(width, height);
-    };
-
-    const handleResize = (): void => {
-      setSize();
-      canv.render();
-    };
-
-    setSize(); // Initial size setting
-    window.addEventListener("resize", handleResize);
-    return () => {
-      window.removeEventListener("resize", handleResize);
-    };
-  }, [canv]);
-
   // RENDERING /////////////////////////////////////////////////
 
+  const makeTitleWithSubtext = (title: string | ReactElement, subtext: string | ReactElement): ReactElement => {
+    return (
+      <>
+        <p style={{ margin: 0 }}>{title}</p>
+        <p style={{ margin: 0, fontSize: "12px" }}>{subtext}</p>
+      </>
+    );
+  };
```

Comment:
1. argument types could be `ReactNode` rather than unioning `ReactElement` with `string`
2. optional suggestion: separate into small utility component? could live at the top of this file e.g.